### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by DamenaM**
### PR Desktop Version

- [ ] adding desktop version
- [ ] adding a contact form 
- [ ] adding cards  
- [ ] adding media queries

### CODE REVIEW

**Review by anagudelogu — COMMENTED** [review]
Hi @DamenaM , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the correct README template. Please use the [updated version](https://github.com/microverseinc/readme-template) and submit your project again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by amejid — COMMENTED** [review]
Hi @DamenaM, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the correct README template. Please use the [updated version](https://github.com/microverseinc/readme-template) and submit your project again. The only optional section is the `FAQ` section. The table of contents should also be present.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -777,25 +816,349 @@ body {
...
+.Facebook360,
+.button2 {
+  display: none;
+}
+
+.Programs-desck {
+  display: none;
+}
+
+@media screen and (min-width: 376px) {
```
Comment (huseyinbiyik) [inline style.css]: Kindly use media query min-width: 768 to the desktop view. Right now, you have no breakpoints for the desktop. And please fix the overflow issue occurring on mobile view. Please check the gif below.
mobile

File: index.html [hunk 2]
```diff
@@ -6,14 +6,17 @@
...
 <body>
 <header class="header">
         <div class="name">
           <a  class="mylogo" href="#">MY logo</a>
         </div>
         <nav class="menu">
           <a href=""><img class="icon-menu" src="./image/logobuger.JPG" alt=""></a>
+          <a class="Portfolio-desc" href="">Portfolio</a>
+          <a class="About-desc" href="">About</a>
+          <a class="contact-desc" href="">Contact</a>
```
Comment (huseyinbiyik) [inline index.html]: Please fix the font-style, size, color, font family and make sure you match the provided Figma design
Your design:
Screenshot 2022-12-16 at 9 59 51 PM

Figma template:
Screenshot 2022-12-16 at 10 10 27 PM

File: index.html [hunk 3]
```diff
@@ -191,10 +240,16 @@ <h4 > LET’S CONNECT</h4>
...
+                <li  href="#"><img  class="Programs-desck" src="./image/descktop/Frame 9 (2).png" alt=""></li>
+            </ul>
             <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
             <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
         </ul>
     </div>
+</div>
 </section>
 <section class="cardcontact">
     <form class="cardcontact1" action="https://formspree.io/f/mwkzayzq" method="post" >
```
Comment (huseyinbiyik) [inline index.html:266]: Kindly make sure your design matches the provided Figma template. Right now there is some alignment issues in both the heading and the paragraph. ↓
Figma template:
Screenshot 2022-12-16 at 10 13 17 PM
Your design:
Screenshot 2022-12-16 at 10 01 54 PM

File: index.html [hunk 4]
```diff
@@ -191,10 +240,16 @@ <h4 > LET’S CONNECT</h4>
...
+                <li  href="#"><img  class="Programs-desck" src="./image/descktop/Frame 11 (1).png" alt=""></li>
+                <li  href="#"><img  class="Programs-desck" src="./image/descktop/Frame 9 (2).png" alt=""></li>
+            </ul>
             <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
             <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
         </ul>
     </div>
+</div>
 </section>
 <section class="cardcontact">
```
Comment (huseyinbiyik) [inline index.html:265]: Kindly fix the gray area and try to follow the Figma template.
Figma template:
Screenshot 2022-12-16 at 10 17 36 PM
Your design:
Screenshot 2022-12-16 at 10 00 54 PM

File: index.html [hunk 5]
```diff
@@ -207,15 +262,17 @@ <h2  class="contactme" >Contact me </h2>
...
                 <input class="texform"  type="email"  id="email"  name="email"   placeholder="Enter Your Email"  required  />
-            </div >
+            </div>
             <div class="form-group2">
-                <textarea class="comment"  name="comment-section" id="comment"  maxlength="500"s  placeholder="Write your message here..."  required ></textarea>
+                <input class="texform2"  name="comment"  id="comment"   cols="50"   rows="6"  placeholder="Write your message here..."  required  />
             </div>
-              
-            <div class="buttonsee2">
+            <div class="buttonsee2" id="contactbuttonid">
```
Comment (huseyinbiyik) [inline index.html]: Please make sure you are following the Figma template for paddings, font-family, font-size, font color for the input element in the form. 
Figma template:
Screenshot 2022-12-16 at 10 22 51 PM
Your design:
Screenshot 2022-12-16 at 10 01 41 PM

File: index.html [hunk 6]
```diff
@@ -207,15 +262,17 @@ <h2  class="contactme" >Contact me </h2>
...
                 <button class="contactbutton"> Get It Touch </button>
```
Comment (huseyinbiyik) [inline index.html]: Kindly make sure if your submit button is aligned well like the provided design.
Your design:
Screenshot 2022-12-16 at 9 59 35 PM
Figma template:
Screenshot 2022-12-16 at 10 26 04 PM

File: index.html [hunk 7]
```diff
@@ -39,19 +42,27 @@ <h4 class="connectmywork_01"> LET’S CONNECT</h4>
     </div>
 </section>
 <section class="gridcontainer">
```
Comment (huseyinbiyik) [inline index.html:50]: Please make sure you add enough margin or padding to the right and left of the card according to the Figma design. 
Figma template:
Screenshot 2022-12-16 at 9 58 42 PM

Your design:
Screenshot 2022-12-16 at 9 59 09 PM

File: index.html [hunk 8]
```diff
@@ -144,45 +188,50 @@ <h3 class="Multi-Post_05" >Multi-Post Stories </h3>
...
-    <div >
-        <h2  class="tonic4" >About Myself </h2>
-        <p class="section1"> Hello I’m a software developer! I can help you build a product , 
-        feature or website Look through some of my work and experience! If you like what you see 
-        and have a project you need coded, don’t hestiate to contact me. </p>
-    </div>
-    <div class = "social-section">
-        <div class="connectmywork">
-            <h4 > LET’S CONNECT</h4>
+<section class="AboutcardBack" id="aboutcardid">
```
Comment (huseyinbiyik) [inline index.html:210]: Kindly fix the design and use flex and make sure your web page responding properly for different screen sizes. Right now, it seems About Myself self is just filling the half of its container.
Figma template:
Screenshot 2022-12-16 at 10 00 30 PM

Your page:
Screenshot 2022-12-16 at 9 59 19 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @DamenaM,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Linter checks passing 🟢
- GitHub flow followed 📂


## Required Changes ♻️
- Kindly use media queries and flexbox to have a responsive web page. Check the gif below and notice that your page is not responding for different screen sizes. All sections have responsivity problems right now. Kindly fix that. 

desktop

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @DamenaM 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**